charts/access/email/README.md

```diff
@@ -2,6 +2,104 @@
 
 This chart sets up and configures a Deployment for the Access Request Email plugin.
 
+## Installation
+
+### Prerequisites
+
+First, you'll need to create a Teleport user and role for the plugin. The following file contains a minimal user that's needed for the plugin to work:
+
+```yaml
+---
+kind: role
+version: v4
```

Comment:
Please double check this as they might already be on v5 now.

Reply:
Right, thanks or catching it!